components/contribution-flow/ContributionFlowHeader.js

```diff
@@ -26,31 +34,36 @@ class NewContributionFlowHeader extends React.Component {
     }).isRequired,
     LoggedInUser: PropTypes.object,
     intl: PropTypes.object,
+    isEmbed: PropTypes.bool,
   };
 
   render() {
-    const { collective } = this.props;
+    const { collective, isEmbed } = this.props;
     const contributors = collective.contributors?.nodes;
 
+    const CollectiveTitleContainer = !isEmbed ? CollectiveTitleLink : Container;
```

Comment:
The problem with this approach is that we will try to set an `href` on a `div` (which probably results in a React/HTML warning?). But we could make this a component:

```es6
const CollectiveTitleContainer = ({ collective, useLink }) => {
  if (useLink) {
    return (...);
  } else {
    return (...);
  }
}
```

Reply:
@Betree : Ops, I somehow didn't think about this; 🤯 . I've refactored the code accordingly. Let me know if you see any issues. 😄